### PR TITLE
Densenet acc

### PR DESCRIPTION
**Pull request by abhinavsp0730**
*Description of changes:*

Fixed densenet accuracy.
By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1097/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1097/

**Comment by abhinavsp0730** [comment]
> 
> 
> The testing accuracy looks off in the case of TF compared to Torch and MXNet. It seems to overfit. Can you please double-check the problem @abhinavsp0730 @terrytangyuan ?
> 
> Tensorflow
> 
> ```python
> loss 0.325, train acc 0.884, test acc 0.715
> ```
> 
> PyTorch
> 
> ```python
> loss 0.153, train acc 0.944, test acc 0.916
> ```
> 
> MXNet
> 
> ```python
> loss 0.146, train acc 0.946, test acc 0.883
> ```
Checked but not able to find why there is a convergence issue, @AnirudhDagar / @terrytangyuan would please have a look?

**Comment by mli** [comment]
Job d2l-en/PR-1097/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1097/

**Comment by abhinavsp0730** [comment]
Hey, @AnirudhDagar  thanks for mentioning the issue.
It's due to a silly indentation mistake.

Please review again.

**Comment by terrytangyuan** [comment]
The accuracy still seems slightly off by 0.05. Could you check again?

**Comment by abhinavsp0730** [comment]
@terrytangyuan I don't think there is any more scope left for doing the changes.
The  . 05 diffrence is due to weights are initialized differently. 
But, there is one way of doing it that we may again run the cl so that
the weights are again initialized differently and we may achieve to overcome 
The 0.5 gap. What  are your thoughts?

**Comment by terrytangyuan** [comment]
What do you think? @mli @AnirudhDagar

**Comment by abhinavsp0730** [comment]
Okay, lemme test my assumption once.

**Comment by AnirudhDagar** [comment]
some stochasticity will always be present. Maybe we'll get different results with CI re-run. Looks ok to me.

**Comment by mli** [comment]
Job d2l-en/PR-1097/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1097/

**Comment by abhinavsp0730** [comment]
Hey, @AnirudhDagar & @terrytangyuan now the gap is negligible 0.004 please review.
